src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -329,9 +329,9 @@ private function getSchema(): Schema
         return $schema;
     }
 
-    public static function formatDateTime(\DateTimeInterface $dateTime)
+    public function formatDateTime(\DateTimeInterface $dateTime): string
```

Comment:
looks like this should should be marked as internal

Reply:
why is this public at all? it should not be public just for the tests.

Reply:
also we should not do this kind of bc break in a patch release

Reply:
How can I avoid the BC break ? leaving the function as it is and create a new one that is private ? :thinking: 

For the tests you're right. I'll change that

Reply:
it's experimental, we can break BC here imho

Reply:
in a patch release?

Reply:
yes, the earlier the better to me :)